lib/synx/xcodeproj_ext/project/object/pbx_group.rb

```diff
@@ -43,32 +55,24 @@ def move_entries_not_in_xcodeproj
         end
 
         def sync_path
-          self.path = basename
+          self.path = work_pathname.relative_path_from(parent.work_pathname).to_s
```

Comment:
Makes sense -- I think! You have a really good handle on this localization stuff, this is great.